src/DigitalProductPassport/LogisticsEmissions_v0.1.py

```diff
@@ -0,0 +1,250 @@
+    emission_intensity: Optional[float] = Field(
+        None,
+        title="Emission Intensity",
+        description="The GHG emission intensity of the road transport "
+        "per transported tonne and kilometer in CO2e grams / tonne / km",
+        example=200,
+    )
+    emissions_per_tce: List[EmissionsPerTCE] = Field(
+        None,
+        title="Emissions per TCE",
+        description="The GHG emissions of the transport chain element related to the road transport leg",
+    )
+
+
+class SeaLeg(CamelCaseModel):
+    leg_identifier: Optional[str] = Field(
+        None,
+        max_length=20,
+        description="The leg identifier",
+        example="7623456365",
+    )
+    origin: Optional[str] = Field(
+        None,
+        max_length=250,
+        title="Origin",
+        description="The location of the transport origin",
+        example="Turku",
+    )
+    destination: Optional[str] = Field(
+        None,
+        max_length=250,
+        title="Destination",
+        description="The location of the transport destination",
+        example="Stockholm",
+    )
+    vessel_type: Optional[VesselType] = Field(
+        None,
+        title="Vessel Type",
+        description="The type of the vessel used for the sea transport",
+        example=VesselType.RO_RO,
+    )
+    freight_condition: Optional[SeaLegFreightCondition] = Field(
+        None,
+        title="Freight Condition",
+        description="The conditions that the cargo is being transported with",
+        example=SeaLegFreightCondition.TEMPERATURE_CONTROLLED,
+    )
+    service_type: Optional[ServiceType] = Field(
+        None,
+        title="Service Type",
+        description="The type of the sea transport service",
+        example=ServiceType.SCHEDULED,
+    )
+    total_emissions: Optional[float] = Field(
+        None,
+        title="Total Emissions",
+        description="The total green house gas (GHG) emissions of the road "
+        "transport and other related logistics hub operations measured in CO2e tonnes",
+        example=7.4,
+    )
+    emission_intensity: Optional[float] = Field(
+        None,
+        title="Emission Intensity",
+        description="The GHG emission intensity of the sea transport per "
+        "transported tonne and kilometer in CO2e grams / tonne / km",
+        example=500,
+    )
+    emissions_per_tce: List[EmissionsPerTCE] = Field(
+        None,
+        title="Emissions per TCE",
+        description="The GHG emissions of the transport chain element related to the road transport leg",
+    )
+
+
+class LogisticsEmissionsDataRequest(CamelCaseModel):
+    product: str = Field(
+        ...,
+        title="Product identifier",
+        description="Technical product identifier used by the manufacturer",
+        example="battery-100wh-s",
+    )
+    id: str = Field(
+        ...,
+        title="Identifier",
+        description="Unique identifier for the product",
+        example="177389-09633",
+    )
```

Comment:
Care to check this from the updates in the spreadhsheet; updates to title, description and examples.